.github/workflows/api-cd.yml

```
@@ -0,0 +1,50 @@
+
+          CONTAINER_ID=$(docker compose ps api --status running --quiet)
+
+          if [ -n "$CONTAINER_ID" ]; then
+            echo "✅ API container is running successfully"
+          else
+            echo "❌ API container is not running"
+            exit 1
+          fi
+
+          echo "📋 Recent api logs:"
+          docker compose logs --tail=10 api
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)